screens/LoadingScreen/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigation } from '@react-navigation/native';
+import firebase from 'firebase';
+import React, { useEffect } from 'react';
+
+interface LoadingScreenProps {
+}
+
+const LoadingScreen: React.FC<LoadingScreenProps> = (props) => {
+  const navigation = useNavigation();
+
+  const resetRoute = (name: string) => (
+    navigation.reset({
+      index: 0,
+      routes: [{ name }],
+    })
+  );
+
+  useEffect(() => {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) resetRoute('Root');
+      else resetRoute('Login');
```

Comment:
Can we avoid using inline statements? 

I don't think the below is any less clean or verbose than inline. In contrast, I think it's better to have curly braces as it is consistent with the rest and in the future what if you decide to do more than just resetRoute.  

```
if (user) { 
    resetRoute('Root'); 
} else { 
    resetRoute('Login');
}
```

Reply:
I think readability is subjective in terms of adding braces; I will avoid using them for short one liners because the superfluous syntax is added noise for nothing, and it impacts my readability of the code. In this case, I think curly braces can be justified because of if/else